packages/xo-web/src/xo-app/backup-ng/new/index.js

```diff
@@ -108,6 +107,11 @@ const REPORT_WHEN_FILTER_OPTIONS = [
 
 const getOptionRenderer = ({ label }) => <span>{_(label)}</span>
 
+const setCheckboxValue = property => (_, { target: { checked } }) => state => ({
+  ...state,
+  [property]: checked,
+})
```

Comment:
You could even reuse the same `effects` by using a `name` property on the `checkbox` element :slightly_smiling_face: